e2e-tests/portal/cypress/e2e/journeys/payment-report-officer/record-corrections/feature-flag-enabled/provide-correction.spec.js

```diff
@@ -144,8 +205,8 @@ context('Provide correction - Fee record correction feature flag enabled', () =>
 
             pendingCorrections.row(1).correctionLink().click();
 
-            provideCorrection.facilityIdInput().should('have.value', '');
-            provideCorrection.reportedFeeInput().should('have.value', '');
+            provideCorrection.facilityIdInput.container().should('have.value', '');
```

Comment:
I'm not sure this selector name change makes sense to me, if it was a container containing the input then why it would have a 'value' doesn't make sense. Could we have `provideCorrection.facilityIdInput` and `provideCorrection.facilityIdError` instead perhaps?